tensorboard/components/tf_paginated_view/test/paginatedViewTests.ts

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+declare function fixture(id: string): void;
+
+describe('tf-paginated-view tests', () => {
+  window.HTMLImports.whenReady(() => {
+    let view: any;
+    beforeEach(() => {
+      view = fixture('paginatedViewFixture');
+      view.limit = 5;
+      view.items = _.range(13);
+    });
+
+    it('renders its subcomponents', (/*done*/) => {
+      //setTimeout(() => {
```

Comment:
I wasn't sure whether it was necessary, but it appears not to be. Will remove. Thanks. :-)